bin/dummy-app

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+
+extension_name="solidus_paypal_commerce_platform"
+
+# Stay away from the bundler env of the containing extension.
+function unbundled {
+  ruby -rbundler -e'b = proc {system *ARGV}; Bundler.respond_to?(:with_unbundled_env) ? Bundler.with_unbundled_env(&b) : Bundler.with_clean_env(&b)' -- $@
+}
+
+# "sqlite" is set by the ORB extension instead of "sqlite3",
+# all other options are already in the format expected by `rails new`.
+test "$DB" = "sqlite" && export DB="sqlite3"
+
+rm -rf ./dummy-app
+unbundled bundle exec rails new dummy-app \
+  --database=${DB:-sqlite3} \
+  --skip-bundle \
+  --skip-git \
+  --skip-keeps \
+  --skip-rc \
+  --skip-spring \
+  --skip-javascript
+
+if [ ! -d "dummy-app" ]; then
+  echo 'dummy-app rails application failed'
+  exit 1
+fi
+
+cd ./dummy-app
+unbundled bundle add solidus --github solidusio/solidus --branch "${BRANCH:-master}" --version '> 0.a'
```

Comment:
I see. It makes sense. That way, we don't need to touch the Gemfile if Solidus updates its version. I thought it was to avoid some cyclic reference issues we had in the past. Thanks for your clarification.